src/components/donation-box.js

```diff
@@ -78,34 +77,41 @@ const Donate = styled.div`
 `
 
 export default class DonationBox extends PureComponent {
+  state = {
+    isMounted: false,
+  }
+
+  componentDidMount() {
+    this.setState({
+      isMounted: true,
+    })
+  }
+
   render() {
-    let currentHref = null
-    let donateURL = null
-    if (typeof window !== 'undefined') {
-      currentHref = window.location.href
-      const search = `utm_source=twreporter.org&utm_medium=article&utm_campaign=${encodeURIComponent(
-        window.location.pathname
-      )}`
+    const { isMounted } = this.state
+    let donateURL = _donateURL
+    if (isMounted) {
       try {
-        const url = new URL(_donatePath)
+        const utmSource = window.location.host
+        const utmCampaign = encodeURIComponent(window.location.pathname)
+        const search = `utm_source=${utmSource}&utm_medium=article&utm_campaign=${utmCampaign}`
+        const url = new URL(donateURL)
         url.search = search
         donateURL = url.toString()
       } catch (e) {
-        donateURL = _donatePath + '?' + search
+        console.warn('Can not get donation url with utm query params', e)
       }
     }
     return (
       <Container>
         <Title>{_content.title}</Title>
         <Text>{_content.desc}</Text>
         <Donate>
-          <ReactGA.OutboundLink
-            eventLabel={`[article_donation_button_click]: ${currentHref}`}
-            to={donateURL || _donatePath}
-            target="_blank"
-          >
-            <p>{_content.bt}</p>
-          </ReactGA.OutboundLink>
+          {isMounted ? (
```

Comment:
Rendering the element when the component is not mounted seems okay. It just take the original `donateURL` without `utm`.